catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -242,13 +265,14 @@ private CreateResponse doCreate(CreateRequest createRequest)
       createResponse = performRemoteCreate(createRequest, createResponse);
 
     } catch (IngestException iee) {
-      INGEST_LOGGER.debug("Ingest error", iee);
       ingestError = iee;
       throw iee;
     } catch (StopProcessingException see) {
+      INGEST_LOGGER.debug("Error during pre ingest on metacards: {}", fileName, see);
       ingestError = see;
       throw new IngestException(PRE_INGEST_ERROR, see);
     } catch (RuntimeException re) {
+      INGEST_LOGGER.debug("Exception during runtime while performing create: {}", fileName, re);
```

Comment:
✏️ I'm realizing my wording was weird in this comment https://github.com/codice/ddf/pull/5144#discussion_r309356135.
I was meaning to not have any debug statements in any of these catch blocks because the finally block is already logging them. i.e. remove this debug statement.

Reply:
fixed